app/src/main/res/values-nb/strings-nb.xml

```diff
@@ -1,501 +1,830 @@
+    <string name="pref_force_wearG5">Tving bruk av Wear innsamlingstjeneste</string>
+    <string name="pref_summary_force_wearG5">Tving telefonen til å bruke Wear innsamlingstjenesten</string>
+    <string name="pref_disable_wearG5_on_lowbattery">Deaktiver tvang ved lavt batteri</string>
+    <string name="pref_summary_disable_wearG5_on_lowbattery">Deaktiver tvang av Wear innsamlingstjeneste ved Lavt batterivarser fra Wear</string>
+    <string name="pref_disable_wearG5_on_missedreadings">Deaktiver tvang ved tapte avlesninger</string>
+    <string name="pref_summary_disable_wearG5_on_missedreadings">Deaktiver Tvang av Wear innsamlingstjenste ved Minutter siden forrige avlesning fra Wear</string>
+    <string name="pref_disable_wearG5_on_missedreadings_level">Minutter siden forrige avlesning fra Wear</string>
+    <string name="pref_node_wearG5">Enhet som kjører innsamlingstjenesten</string>
+    <string name="pref_summary_node_wearG5">Tving telefonen til å bruke denne Wear innsamlingstjenesten</string>
+
+    <string name="pref_sync_wear_logs">Synkroniser Wear Logger</string>
+    <string name="pref_summary_sync_wear_logs">Send Wear logger til hendelseslisten</string>
+    <string name="pref_wear_logs_prefix">Wear Logg Prefiks</string>
+    <string name="pref_summary_wear_logs_prefix">Prefiks brukes for Wear loggoppføringer</string>
+    <string name="pref_show_treatments">Vis Behandlinger</string>
+    <string name="pref_summary_show_treatments">Vis behandlingspunkt på grafen</string>
+    <string name="pref_use_wear_health">Bruk Wear helsedata</string>
+    <string name="pref_summary_use_wear_health">Samle inn og vis pedometer data hvis tilgnengelig</string>
+
+    <string name="pebble_integration">Pebble Integrasjon</string>
+    <string name="send_data_to_pebble_watchface">Send data til Pebble urskive.</string>
+    <string name="pebble_watch_integration">Pebble Klokke Integrasjon</string>
+    <string name="standard_or_trend_pebble_watchface">Standard eller Trend pebble urskive</string>
+    <string name="choose_pebble_watchface">Velg Pebble urskive</string>
+    <string name="message_to_display_when_bgl_hits">Melding som skal vises når BGL treffer spesiell verdi over</string>
+    <string name="text_to_display_when_hitting_special_value">Tekst som skal vises når du treffer spesiell verdi</string>
+    <string name="special_value">Spesiell verdi</string>
+    <string name="history">Historikk</string>
+    <string name="statistics">Statistikk</string>
+    <string name="snooze_alert">Utsett Alert</string>
+    <string name="less_common_settings">Mindre vanlige innstillinger</string>
+    <string name="view_recent_errors_warnings">Vis nylige Feil / Advarsler</string>
+    <string name="other_settings">Andre innstillinger</string>
+    <string name="older_gratuitous_wakelocks">Eldre umotiverte wakelocks som kan tappe batteriet ekstra, men kan være nødvendig for Nightwatch eller Android Wear</string>
```

Comment:
"Umotiverte" seems incorrect, but I am uncertain of the correct translation here

Reply:
Trying with "unødvendige". google explains gratuitous as "done without good reason; uncalled for."